spec/presenters/finder_presenter_spec.rb

```diff
@@ -82,6 +93,28 @@
     end
   end
 
+  describe "national applicability" do
+    it "has applicable nations in the metadata if it is only applicable to some nations" do
+      applies_to = national_applicability_presenter.page_metadata[:other]["Applies to"]
+      expect(applies_to).to include("England", "Northern Ireland", "Scotland", "Wales")
+    end
+
+    it "has no applicable nations in the metadata if it applies to all nations" do
+      applies_to = government_presenter.page_metadata
+      expect(applies_to).not_to have_key(:other)
+    end
+
+    it "sets rel='external' for an external link" do
+      href = "http://www.externalwebsite.com"
+      expect(national_applicability_presenter.send(:is_external?, href)).to be(true)
```

Comment:
Testing private methods is a bit of a smell. I think a test that examines the output fragment and checks that the link contains the `rel="external"` attribute would be more useful and valuable.